api/src/handlers.rs

```diff
@@ -85,26 +85,28 @@ pub struct Member {
 
 #[derive(Serialize, Deserialize, Clone, Debug, FromRow)]
 #[allow(non_snake_case)]
```

Comment:
do we still need this?

Reply:
No, I forgot to check if there were any other suppressed warnings.